src/Layer/ColorLayer.js

```diff
@@ -47,19 +49,6 @@ class ColorLayer extends Layer {
      *
      * // Add the layer
      * view.addLayer(color);
-     *
```

Comment:
why you remove example?

Reply:
Because this type of adding a layer is going to be deprecated (if I understand correctly), so no need for this example to stay.

Reply:
the example could be rewritten?